nginx.conf

```diff
@@ -105,6 +105,9 @@
     rewrite ^/(user/password/send-reset-email)$ $scheme://$http_host/graph/$1;
     rewrite ^/(login)$ $scheme://$http_host/graph/$1; 
 
+    # Remove next line if bug is fixed https://github.com/grafana/grafana/issues/27588
```

Comment:
@tiagomotasantos do we have PMM tickets for this revert and for the one above?

Reply:
https://jira.percona.com/browse/PMM-6535

(for the one above I don't know, searched but couldn't find it)